tracing-appender/src/rolling/builder.rs

```diff
@@ -181,6 +183,33 @@ impl Builder {
         Self { suffix, ..self }
     }
 
+    /// Keep the last `n` log entries on disk.
+    ///
+    /// # Examples
+    ///
+    /// Setting a suffix:
+    ///
+    /// ```
+    /// use tracing_appender::rolling::RollingFileAppender;
+    ///
+    /// # fn docs() {
+    /// let appender = RollingFileAppender::builder()
+    ///     .keep_last_n_logs(5) // only the most recent 5 logs files will be kept
+    ///     // ...
+    ///     .build("/var/log")
+    ///     .expect("failed to initialize rolling file appender");
+    /// # drop(appender)
+    /// # }
+    /// ```
+    ///
+    /// If no value is supplied, `RollingAppender` will not remove any files.
+    pub fn keep_last_n_logs(self, n: usize) -> Self {
```

Comment:
take it or leave it: maybe a clearer name for this would be `max_log_files`? what do you think?

Reply:
this is definitely a better name, taking it :)

Reply:
I'm also changing `keep_last` to `max_files`, we can always revert it you think `keep_last` was a better name.